client/js/app.js

```diff
@@ -73,7 +73,7 @@ function initWithRaven() {
 // If the DOM is not loading, we can init, else wait till we do
 const domNotLoading = document.readyState !== 'loading';
 if (domNotLoading) {
-  initWithRaven();
+  init();
```

Comment:
Hmm. I don't like this as then we don\t get error handling.
How can we make sure raven is available?